crates/steel-core/src/rerrs.rs

```diff
@@ -240,7 +241,13 @@ impl SteelErr {
     pub fn emit_result(&self, file_name: &str, file_content: &str) {
         // let opts = Opts::();
         // let config = codespan_reporting::term::Config::default();
-        let writer = StandardStream::stderr(ColorChoice::Always);
+        let color = if std::io::stderr().is_terminal() {
+            ColorChoice::Auto
+        } else {
+            ColorChoice::Never
+        };
```

Comment:
:information_source:

* Don't colorize output if stderr is not a terminal.
* Use `Auto` instead of `Always` to respect `NO_COLOR` and such.